src/averageAges.js

```diff
@@ -15,6 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = century
+    ? people.filter(person => person.sex === 'm'
```

Comment:
Consider using more descriptive variable names, like `menArray` instead of `arrayOfMen`. 🤔